src/model/decorators/DraftDecoratorType.js

```diff
@@ -25,7 +26,7 @@ export type DraftDecoratorType = {
   /**
    * Given a `ContentBlock`, return an immutable List of decorator keys.
    */
-  getDecorations(block: ContentBlock): List<?string>,
+  getDecorations(contentState: ContentState, block: ContentBlock): List<?string>,
```

Comment:
I'm thinking it may make sense to change the second argument to be a key string. That way we would never run the risk of passing in a `ContentBlock` that is not part of the `ContentState`. It's a breaking API change either way, so we might as well just totally break it. :)